app/assets/javascripts/app/components/discussion_card/templates/discussion_card.html.slim

```diff
@@ -1,20 +1,20 @@
-md-card class="Discussion Discussion--{{ ctrl.discussion.status }}"
-  a.Discussion-link ui-sref="discussions.show({id: ctrl.discussion.id})"
-    .Discussion-title
+md-card class="DiscussionCard DiscussionCard--{{ ctrl.discussion.status }}" ng-class="{ 'DiscussionCard--closed': !ctrl.discussion.open }"
```

Comment:
After you make this multiline it can be merged :+1: btw, give meaningful titles to pull requests